docs/content/configuration/storage.md

```diff
@@ -133,7 +134,7 @@ After this you can pass you credentials inside `config.toml` file
 
 ## Minio
 
-[Minio](https://www.minio.io/) is an open source object storage server. If you never used minio you can read this [quick start guide](https://docs.minio.io/)
+[Minio](https://www.minio.io/) is an open source object storage server that provides an interface for S3 compatible block storages. If you have never used minio, you can read this [quick start guide](https://docs.minio.io/). Any block storage that can use the S3 compatible block storage supported through the minio interface is supported by Athens. Below, you can find different configuration options we provide for Minio. Example configuration for Digital Ocean and Alibaba OSS block storages have also been provided below.
```

Comment:
Maybe: Any s3 compatible object storage is supported by Athens through the minio interface. Below,...
...Example configuration for Digital Ocean Spaces and Alibaba OSS storages are provided below.

I'm not a native speaker so take it with a grain of salt.

Reply:
@marpio Shorter sentences and active voice are always better. I have taken in your suggestions. Thank you!